src/app/actions/get-messages.ts

```diff
@@ -0,0 +1,27 @@
+"use server";
```

Comment:
@alhonaut  Use API Routes when fetching data, and Server Actions when performing actions/mutations.

Reply:
For now it is fine. Leave it like this.